static/operator-profile-sds.yaml

```diff
@@ -5,6 +5,5 @@ metadata:
   namespace: istio-operator
   name: example-istiocontrolplane
 spec:
-  profile: minimal
```

Comment:
This should be sds?

Reply:
the automated tooling does some seding here, but it appears I didn't revert enough PRs - will address.

Reply:
this should not be sds, the name of the file is minimal?

Reply:
the old file name is minimal but it's being changed back to operator-profile-sds.yaml.